GitUI/Plugin/PluginRegistry.cs

```diff
@@ -22,18 +29,23 @@ public static void Initialize()
                     return;
                 }
 
-                ManagedExtensibility.SetApplicationDataFolder(AppSettings.ApplicationDataPath.Value);
-
-                foreach (var plugin in ManagedExtensibility.GetExports<IGitPlugin>().Select(lazy => lazy.Value))
+                try
                 {
-                    plugin.SettingsContainer = new GitPluginSettingsContainer(plugin.Name);
-
-                    if (plugin is IRepositoryHostPlugin repositoryHostPlugin)
+                    foreach (var plugin in ManagedExtensibility.GetExports<IGitPlugin>().Select(lazy => lazy.Value))
                     {
-                        GitHosters.Add(repositoryHostPlugin);
-                    }
+                        plugin.SettingsContainer = new GitPluginSettingsContainer(plugin.Name);
+
+                        if (plugin is IRepositoryHostPlugin repositoryHostPlugin)
+                        {
+                            GitHosters.Add(repositoryHostPlugin);
+                        }
 
-                    Plugins.Add(plugin);
+                        Plugins.Add(plugin);
+                    }
+                }
+                catch
+                {
+                    // no-op
```

Comment:
❓ Why?

Reply:
This try-catch was originally in `FormBrowse.cs`. 
@RussKie moved it here in 7d30a22aba4478d8b231bf230a68cfcf9b43c83b.

Reply:
If a plugin fails to load we don't want to crash the app because of it.